indico/modules/events/sessions/util.py

```diff
@@ -0,0 +1,34 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.modules.events.sessions.models.sessions import Session
+
+
+def create_session(event, wtform=None, session_config=None, base_session=None):
```

Comment:
The idea here was to not rely on a WTForm. So you'd pass the `form.data` dict.  Also `create_session` to modify a session is extremely confusing. I'd rather have two functions and a `_set_session_data` like function called from both of them.